backend/src/app.js

```diff
@@ -6,11 +6,21 @@ const mongoose = require('mongoose');
 const session = require('express-session');
 const cookieParser = require('cookie-parser');
 const MongoStore = require('connect-mongo')(session);
+const cors = require('cors');
 
 const auth = require('./auth');
 const api = require('./routes');
 const app = express();
 
+app.use(
+  cors({
```

Comment:
I'd be careful about enabling CORS for all requests.

> If you are using authentication based on session cookies, you probably shouldn’t allow CORS requests by everyone. A malicious website can issue e-mail sending requests to api.yoursebsite.com via an AJAX request without the specific permission of your user.

https://mobilejazz.com/blog/which-security-risks-do-cors-imply/

Reply:
Gotcha -- this was added from a tutorial I was following earlier, but I don't think I need it anymore... Going to remove it and test the auth flow. As long as it all works w/o it, I don't think we need this.

Reply:
Ended up removing `cors` from this file.